core/node/api_server/src/web3/state.rs

```diff
@@ -106,6 +106,7 @@ pub struct InternalApiConfig {
     pub state_transition_proxy_addr: Option<Address>,
     pub transparent_proxy_admin_addr: Option<Address>,
     pub user_facing_diamond_proxy_addr: Address,
+    pub diamond_proxy_addr: Address,
```

Comment:
Can you please explain what's the difference between `user_facing_diamond_proxy_addr` and `diamond_proxy_addr`?